crates/re_data_ui/src/entity_db.rs

```diff
@@ -60,6 +61,14 @@ impl crate::DataUi for EntityDb {
                 ui.label(store_source.to_string());
                 ui.end_row();
 
+                if let Some(store_version) = store_version {
+                    re_ui.grid_left_hand_label(ui, "Protocol Version");
```

Comment:
Nit: we also use protocol version to define the tcp stream protocol as in:

https://github.com/gurry/rerun/blob/ae9aeedf76a425d07f8a8064f4a0a4436c9ae797/crates/re_sdk_comms/src/lib.rs#L22

Can we call this this something like the "SDK_Version"

Reply:
I used to call it `RRD Version`, I can revert to that -- `SDK Version` is already quite overloaded

Reply:
`Source` and `Source RRD version` it is